locales/it/index.yml

```diff
@@ -1259,7 +1259,7 @@ wallet:
     cancelPaymentSuccess: Il pagamento di questo avviso è stato annullato, potrai pagarlo nuovamente tra qualche minuto
     accessibility:
       edit: "tocca per modificare"
-      transactionCosts: "{{cost}}, transazione gestita da {{psp}}"
+      transactionCosts: "{{cost}}, {{psp}}"
```

Comment:
@dgopsq I think the issue is that the string "psp" contains the words "Gestita da", that's why the VoiceOver reads "transazione gestita da gestita da".